scripts/convert.sh

```diff
@@ -0,0 +1,17 @@
+#!bin/bash
+cd ../../cloud/aggregation/cache
```

Comment:
change the folder to `FedScale/scriipts/install-mnn`. Please remember other paths that cites it.